transport/whisper/whisper_service.go

```diff
@@ -458,236 +386,3 @@ type RetryConfig struct {
 	StepTimeout time.Duration
 	MaxRetries  int
 }
-
-func (a *WhisperServiceTransport) requestMessagesWithRetry(conf RetryConfig, r MessagesRequest) (MessagesResponse, error) {
```

Comment:
Why do we get rid of this? Console Client relies on these methods.

Reply:
not sure, I believe this might be the old `history` stuff that has been removed from status-console-client (we need to fetch the correct topics), it seems to be compiling ok with this version, do you want to keep it even if it's not currently used?

Reply:
Oh sorry, I forgot. Mailserver support was removed at some point when we started moving stuff around.